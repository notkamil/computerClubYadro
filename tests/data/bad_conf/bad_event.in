3
08:00 20:00
10
So you're a tough event
Like it really rough event
Just can't get enough event
Chest always so puffed event
I'm that bad type
Make your mama sad type
Make your girlfriend mad tight
Might seduce your dad type
I'm the bad event
Duh