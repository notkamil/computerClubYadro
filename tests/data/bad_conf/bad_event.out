So you're a tough event